build.sbt

```diff
@@ -478,6 +478,20 @@ lazy val icebergShaded = (project in file("icebergShaded"))
     // Make the 'compile' invoke the 'assembly' task to generate the uber jar.
   )
 
+lazy val hudi = (project in file("hudi"))
```

Comment:
please add `hudi` to `sparkGroup` 

https://github.com/delta-io/delta/blob/80a00a9fa50868573f52c31ee066439515907713/build.sbt#L1131

Reply:
Updated